src/pkg/jobmonitor/redis.go

```diff
@@ -102,6 +107,20 @@ func (r *redisClientImpl) StopPendingJobs(ctx context.Context, jobType string) (
 	return jobIDs, nil
 }
 
+// removeJobStatusInRedis remove job status track information from redis, to avoid performance impact when the jobIDs is too large, use batch to remove
+func (r *redisClientImpl) removeJobStatusInRedis(ctx context.Context, jobIDs []string) {
+	conn := r.redisPool.Get()
+	defer conn.Close()
+	for _, id := range jobIDs {
+		redisKeyStatus := rds.KeyJobStats(fmt.Sprintf("{%s}", r.namespace), id)
```

Comment:
In addition to job_stats, there are residual job IDs in `{harbor_job_service_namespace}:job_track:inprogress `.

Reply:
Updated